### PR TITLE
Instructor: feedback results: provide copy instructions for Mac OS #7350

### LINKED ISSUE #7350: Instructor: feedback results: provide copy instructions for Mac OS
In instructorFeedbackResultsPage, when using the "click here" text to view the results in a table, instructions are provided to copy and paste the table into an Excel workbook.
feedbacktable
However, Mac systems use `Command + C` and `Command + V` instead, which could be added to the instructions.

### PR DESCRIPTION
**Pull request by umangborad123**
Fixes #7350 

**PR Checklist**

Ensure that you have:
- [x] Read and understood our PR guideline: https://github.com/TEAMMATES/teammates/blob/master/docs/process.md#step-4-submit-a-pr
  - [x] Added the issue number to the "Fixes" keyword above
  - [x] Titled the PR as specified in the abovementioned document
- [x] Made your changes on a branch other than `master` and `release`
- [x] Gone through all the changes in this PR and ensured that:
  - [x] They addressed one (and only one) issue
  - [x] No unintended changes were made
- [x] Run and passed static analysis: `./gradlew lint` and `npm run lint`
- [x] Added/updated tests, if changes in functionality were involved
- [x] Added/updated documentation to public APIs (classes, methods, variables), if applicable

**Outline of Solution**

### CODE REVIEW

File: src/test/resources/pages/instructorFeedbackResultsPageOpen.html [hunk 1]
```diff
@@ -199,11 +199,11 @@
           <span class="help-block" style="display:inline;">
             Tips: After selecting the table,
             <kbd>
-              Ctrl + C
+              Ctrl + C or Command + C
```
Comment (aashil) [inline src/test/resources/pages/instructorFeedbackResultsPageOpen.html]: Please change this to just Command + C. Same with paste.

Comment (thenaesh) [inline src/test/resources/pages/instructorFeedbackResultsPageOpen.html]: What about Windows/Linux users?

Comment (aashil) [inline src/test/resources/pages/instructorFeedbackResultsPageOpen.html]: Apologies. I glanced the issue and jumped to the conclusion that this file is related to just macOS when clearly it's general instruction for all platforms.

Comment (umangborad123) [inline src/test/resources/pages/instructorFeedbackResultsPageOpen.html]: I don't understand why is it failing Travis Cl

**Comment by chowyb** [comment]
@umangborad123 you will need to use [GodMode](https://github.com/TEAMMATES/teammates/blob/master/docs/godmode.md) to update the failing tests.

**Comment by umangborad123** [comment]
This is my first time and I am not sure I follow what you said. Can you be more specific, please?

**Comment by chowyb** [comment]
@umangborad123
When running tests on Travis, you can see which tests failed when clicking on the red 'x' beside the commit. You can then run GodMode on those tests to update the HTML files so that they pass the comparison.

**Comment by umangborad123** [comment]
@chowyb Hi, I enabled godmode by setting `isGodModeEnabled` in `BaseUiTestCase` to `true`. 

`Failed suite [Failed suite [ci-tests]] > parallel-ui-tests(failed)(failed) > teammates.test.cases.browsertests.InstructorFeedbackResultsPageUiTest.testHtmlContent FAILED
    java.lang.AssertionError at InstructorFeedbackResultsPageUiTest.java:76
1 test completed, 1 failed
There were failing tests
:killFirefox3
:testTry4
java.lang.AssertionError: Please append a unique id (e.g your name) to each of the default account in test.properties in order to use God mode, e.g change alice.tmms to alice.tmms.<yourName>, charlie.tmms to charlie.tmms.<yourName>, etc.
	at org.testng.AssertJUnit.fail(AssertJUnit.java:59)
	at teammates.test.driver.TestProperties.verifyReadyForGodMode(TestProperties.java:142)
	at teammates.test.pageobjects.AppPage.regenerateHtmlFile(AppPage.java:886)
	at teammates.test.pageobjects.AppPage.testAndRunGodMode(AppPage.java:877)
	at teammates.test.pageobjects.AppPage.verifyHtmlPart(AppPage.java:861)
	at teammates.test.pageobjects.AppPage.verifyHtml(AppPage.java:824)
	at teammates.test.cases.browsertests.InstructorFeedbackResultsPageUiTest.testContent(InstructorFeedbackResultsPageUiTest.java:76)
	at teammates.test.cases.browsertests.InstructorFeedbackResultsPageUiTest.testHtmlContent(InstructorFeedbackResultsPageUiTest.java:43)
	...
Failed suite [Failed suite [Failed suite [ci-tests]]] > parallel-ui-tests(failed)(failed)(failed) > teammates.test.cases.browsertests.InstructorFeedbackResultsPageUiTest.testHtmlContent FAILED
    java.lang.AssertionError at InstructorFeedbackResultsPageUiTest.java:76
1 test completed, 1 failed
:testTry4 FAILED
:killFirefox4
FAILURE: Build failed with an exception.
What went wrong:
Execution failed for task ':testTry4'.
There were failing tests`

**Comment by chowyb** [comment]
@umangborad123 it looks like you will need to append a unique id (such as your name) to each of the default accounts in test.properties, which should have been covered in paragraph 4 of [this step](https://github.com/TEAMMATES/teammates/blob/master/docs/settingUp.md#step-3-set-up-project-specific-settings-and-dependencies).

**Comment by umangborad123** [comment]
I append unique id like you said, but still no luck! Can you help me one last time? What exactly I am supposed to be doing?

`Failed suite [ci-tests] > parallel-ui-tests(failed) > teammates.test.cases.browsertests.InstructorFeedbackResultsPageUiTest.testHtmlContent FAILED
    java.lang.AssertionError at InstructorFeedbackResultsPageUiTest.java:76
1 test completed, 1 failed
There were failing tests
:killFirefox2
:testTry3
Failed suite [Failed suite [ci-tests]] > parallel-ui-tests(failed)(failed) > teammates.test.cases.browsertests.InstructorFeedbackResultsPageUiTest.testHtmlContent FAILED
    java.lang.AssertionError at InstructorFeedbackResultsPageUiTest.java:76
1 test completed, 1 failed
There were failing tests
:killFirefox3
:testTry4
java.lang.AssertionError: Student MOTD URL defined in app.student.motd.url in build.properties must not be empty. It is advised to use test-student-motd.html to test it.
	at org.testng.AssertJUnit.fail(AssertJUnit.java:59)
	at teammates.test.driver.TestProperties.verifyReadyForGodMode(TestProperties.java:147)
	at teammates.test.pageobjects.AppPage.regenerateHtmlFile(AppPage.java:886)
	at teammates.test.pageobjects.AppPage.testAndRunGodMode(AppPage.java:877)
	at teammates.test.pageobjects.AppPage.verifyHtmlPart(AppPage.java:861)
	at teammates.test.pageobjects.AppPage.verifyHtml(AppPage.java:824)
	at teammates.test.cases.browsertests.InstructorFeedbackResultsPageUiTest.testContent(InstructorFeedbackResultsPageUiTest.java:76)
	at teammates.test.cases.browsertests.InstructorFeedbackResultsPageUiTest.testHtmlContent(InstructorFeedbackResultsPageUiTest.java:43)
	...
Failed suite [Failed suite [Failed suite [ci-tests]]] > parallel-ui-tests(failed)(failed)(failed) > teammates.test.cases.browsertests.InstructorFeedbackResultsPageUiTest.testHtmlContent FAILED
    java.lang.AssertionError at InstructorFeedbackResultsPageUiTest.java:76
1 test completed, 1 failed
:testTry4 FAILED
:killFirefox4
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':testTry4'.
> There were failing tests
* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.
BUILD FAILED
Total time: 23 mins 19.478 secs
The command "./gradlew appengineRun ciTests" exited with 1.
after_failure.1
1.07s$ gem install gist
Fetching: gist-4.6.1.gem (100%)
Successfully installed gist-4.6.1
1 gem installed
after_failure.2
11.82s$ ./gradlew generateFailedCmd
:generateFailedCmd
Run failed tests locally by downloading the XML file containing the failed tests using the command:
Windows: gradlew.bat -Pgist=a34e89c531747b0c70964183ddf30ae7 downloadFailedXml
Linux/OS X: ./gradlew -Pgist=a34e89c531747b0c70964183ddf30ae7 downloadFailedXml
followed by running "Failed tests" in Eclipse/IntelliJ or using the following command:
Windows: gradlew.bat failedTests
Linux/OS X: ./gradlew failedTests
BUILD SUCCESSFUL
Total time: 11.417 secs
after_script
10.98s$ ./gradlew appengineStop
:appengineDownloadSdk
:appengineStop
BUILD SUCCESSFUL
Total time: 10.604 secs
``

**Comment by chowyb** [comment]
@umangborad123 it looks like you'll have to set `app.student.motd.url = test-student-motd.html` in `build.properties`, as mentioned in the error message. While this is not currently covered in the setup instructions, it is being fixed by #7362.

**Pull request closed without merging**